src/middlewares/auth.middleware.ts

```diff
@@ -1,13 +1,13 @@
 import { Injectable, NestMiddleware } from "@nestjs/common";
-import { verify } from "jsonwebtoken";
+import { JwtPayload, verify } from "jsonwebtoken";
 import { NextFunction, Response } from "express";
 import { UserService } from "src/api/users/user.service";
 import { ExpressRequestInterface } from "src/types/expressRequest.interface";
 
 @Injectable()
 export class AuthMiddleware implements NestMiddleware {
     constructor(private readonly userService: UserService) {}
-    async use(req: ExpressRequestInterface, res: Response, next: NextFunction) {
+    async use(req: ExpressRequestInterface, _res: Response, next: NextFunction) {
```

Comment:
If we give the arguments of the query to the lower slash, then it is worth giving everything